rama-http-types/src/curl/body.rs

```diff
@@ -0,0 +1,16 @@
+use crate::{Body, Request};
+use rama_core::error::OpaqueError;
+
+pub fn request_to_curl_command<T: IntoCurlCommand>(request: T) -> Result<(String, Request), OpaqueError> {
+    todo!()
```

Comment:
Well organised. Thanks for that. FYI you probably can make use of `request_headers_to_curl_command` here (or some common internal methods at the least) to do most of the heavy lifting for you, except for the body (if there's any of course).